src/api/Flooq.Test/Controllers/ContactControllerTest.cs

```diff
@@ -0,0 +1,175 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Flooq.Api.Controllers;
+using Flooq.Api.Models;
+using Flooq.Api.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Flooq.Test.Controllers;
+
+[TestClass]
+public class ContactControllerTest
+{
+  private readonly Mock<IContactService> _contactServiceMock = new();
+  private readonly Contact _contact = new()
+  {
+    Email = "test@example.com"
+  };
+  private readonly Random _random = new();
+  private int _n;
+
+  [TestInitialize]
+  public void Setup()
+  {
+    _n = _random.Next(2, 11);
+  }
```

Comment:
Nice idea, but is this really needed?

Reply:
I think it's an easy way to test multiple operations.